mlops/nyc_taxi/src/mlops_pipeline.py

```diff
@@ -192,7 +193,7 @@ def execute_pipeline(
             pipeline_job, experiment_name=experiment_name
         )
 
-        print(f"The job {pipeline_job.name} has been submitted!")
+        print(f"Pipeline job '{pipeline_job.name}' has been submitted successfully!")
```

Comment:
Minor comment.. this is different from the london_taxi line `print(f"The job {pipeline_job.name} has been submitted!")`

Reply:
Made this change so they match